tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -137,13 +137,16 @@ build_with_container() {
 	image_dir=$(readlink -f "$(dirname "${image}")")
 	image_name=$(basename "${image}")
 
-	REGISTRY_ARG=""
+	engine_build_args=""
```

Comment:
I get `docker_run_args`, any particular reason for renaming `REGISTRY_ARG`?

Reply:
Just instead of having two variables that contains build args i unified all args into single variable, name has changed as it's not necessarily contains only registry related arguments